pronto/api/entry/signatures.py

```diff
@@ -385,3 +387,19 @@ def get_signatures_annotations(accession):
     cur.close()
     con.close()
     return jsonify(signatures)
+
+
+def check_annotation(acclist, cur):
+    search_acc = ['%' + acc + '%' for acc in acclist]
+    cur.execute(f"""SELECT A.ANN_ID, max(S.CT)
+    FROM INTERPRO.COMMON_ANNOTATION A
+    INNER JOIN INTERPRO.ENTRY2COMMON E ON A.ANN_ID = E.ANN_ID
+    LEFT OUTER JOIN (SELECT ANN_ID, COUNT(*) CT
+    FROM INTERPRO.ENTRY2COMMON GROUP BY ANN_ID) S ON A.ANN_ID = S.ANN_ID
+    WHERE {' OR '.join(["A.TEXT LIKE :1"] * len(acclist))}
+    GROUP BY A.ANN_ID""""", search_acc)
+    accessions = []
+    for each in cur.fetchall():
+        ann_id = each[0]
+        accessions.append(ann_id)
+    return accessions
```

Comment:
Use list comprehension:

```suggestion
    return [row[0] for row in cur.fetchall()]
```